### DISCUSSION TITLE
YAML config for only Current readings?

### DISCUSSION THREAD

**Discussion by arknoto** [emporia-vue-local/esphome#230] [Q&A] [answered]
Can anyone provide an example configuration with only the CT clamp current readings?  I want to get the current and voltage readings to Node-Red and do all of the math for wattages there.

**Comment by flaviut** [comment]
See https://github.com/emporia-vue-local/esphome/discussions/225#discussioncomment-7380649

You can then mark your wattage sensors as `internal: true` so they don't show up.

edit: in short, create a bunch of entries like the following, matching your phase config:

```
  - platform: template
    name: "Circuit 9 Current"
    lambda: return id(cir9).state / id(phase_a_voltage).state;
    id: cir9_current
    device_class: current
    state_class: measurement
    unit_of_measurement: "A"
```

**Reply by arknoto** [reply, marked as answer]
Thank you, I am still having trouble getting this config working.  If anyone has an example config that uses MQTT only and outputs only current readings.
